File: Spicy snack/Agecalculator.py
fathers_current_age = int(input("Enter father's current age: "))
sons_current_age = int(input("Enter son's current age: "))
years_ago = 2 * sons_current_age - fathers_current_age


if (fathers_current_age or sons_current_age > 0) and (fathers_current_age or sons_current_age < 81):
    print(abs(years_ago))
else:print("Enter an age between 1 and 80")

File: Spicy snack/AgeCalculator.java
import java.util.Scanner;
public class AgeCalculator{
public static void main(String [] args){
Scanner input = new Scanner(System.in);
System.out.println("Enter father's age");
int fathersAge = input.nextInt();
System.out.println("Enter son's age");
int sonsAge = input.nextInt();
int yearsAgo = ( 2 * sonsAge) - fathersAge;
if(fathersAge > 0  && fathersAge < 81 || sonsAge > 0 && sonsAge < 81){
System.out.printf("%d",yearsAgo);
}else{
System.out.print("Enter an age between 1 and 80");
}
}
}
